web-api/src/persistence/ses/getSesStatus.ts

```diff
@@ -1,15 +1,21 @@
+import { GetSendStatisticsCommand, type SESClient } from '@aws-sdk/client-ses';
 export const getSesStatus = async ({
   applicationContext,
 }: {
   applicationContext: IApplicationContext;
 }): Promise<boolean> => {
-  const SES = applicationContext.getEmailClient();
+  const sesClient: SESClient = applicationContext.getEmailClient();
   const HOURS_TO_MONITOR = 24;
-  const { SendDataPoints } = await SES.getSendStatistics({}).promise();
+  const cmd = new GetSendStatisticsCommand();
+  const { SendDataPoints } = await sesClient.send(cmd);
   const numberOfDataPoints = HOURS_TO_MONITOR * 4; // each data point is a 15 minute increment
-  const sesHealth = SendDataPoints.slice(0, numberOfDataPoints).every(
+  const sesHealth = SendDataPoints?.slice(0, numberOfDataPoints).every(
     ({ Rejects }) => Rejects === 0,
   );
-
+  //todo: what is the meaning of SendDataPoints being undefined.  Can we assume success? Failure?
```

Comment:
Is this todo meant to be here?

Reply:
Mostly no, but kinda yes?  It was a note to self until I figured out the answer, but I forgot to remove it. That said, I still don't know the answer.